### PR TITLE
Add debugging for scale e2e test errors

### PR DESCRIPTION
**Pull request by liggitt**
xref https://github.com/kubernetes/kubernetes/issues/64450#issuecomment-400124285

### CODE REVIEW

**Comment by liggitt** [comment]
/assign @BenTheElder @luxas

will keep an eye on the kubeadm test after this merges to see what the debug output reveals

**Review by BenTheElder — APPROVED** [review]
/lgtm
/hold
very curious to see what was actually going on with this test

**Pull request merged**
